packages/form-builder/src/components/canvas/formBlocks.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+    BlockList,
+    BlockTools,
+    ObserveTyping,
+    WritingFlow,
+    ButtonBlockAppender,
+} from "@wordpress/block-editor";
+
+const Component = () => {
```

Comment:
For consistency, can you please rename this file to `FormBlocks.tsx` and the component to `FormBlocks`

Also, with the current setup you'll probably need to do `import * as React from 'react';` in typescript.